packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js

```diff
@@ -452,6 +452,10 @@ export default class IpListController {
         );
     };
 
+    $scope.displayStatistics = function displayStatistics(ipBlock) {
+      self.goToStatistics(ipBlock);
+    };
```

Comment:
No need to manage tracking anymore ? As before you managed it directly in the template 
`data-ng-click="setAction('ip/mitigation/statistics/ip-ip-mitigation-statistics', { ipBlock: ipBlock, ip: ip ,tracking : tracking['statistics']})"`

If not maybe you can simplify as 

```suggestion
    $scope.displayStatistics = self.goToStatistics;
```

Reply:
For the moment the tracking is not set for the new page Network Security Dashboard.